website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Sign API calls with managed keys fail
+
+#### Affected versions
+
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+The Transit sign API call responds with an error "requested version for signing does not contain a private part" when
+a Transit sign request uses a managed key.
+
+<Note>Signing APIs are not affected with non-managed key, key types</Note>
```

Comment:
```suggestion
<Note>

The issue does not affect signing with the following key types:

- non-managed keys

</Note>
```

Style correction: make key information easy to scan

Reply:
Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b, just tweaked non-managed keys to Transit native keys